gunicorn/glogging.py

```diff
@@ -50,11 +45,11 @@
         }
 
 
-CONFIG_DEFAULTS = dict(
-        version=1,
-        disable_existing_loggers=False,
+CONFIG_DEFAULTS = {
```

Comment:
The `dict(...)` version is pretty readable. I don't think we should do cosmetic-only changes.

Reply:
✅ by reverting 4aa3a5e44c26dbf269ad9cd637d61c46fad52d52.